modules/install-sync-gateway/install-sync-gateway

```diff
@@ -257,12 +257,12 @@ function install {
 
   log_info "Starting Sync Gateway install..."
 
-  if os_is_ubuntu "16.04"; then
+  if os_is_ubuntu "16.04" || os_is_ubuntu "18.04"; then
     install_sync_gateway_on_ubuntu "$edition" "$version" "$checksum" "$checksum_type"
   elif os_is_amazon_linux "2"; then
     install_sync_gateway_on_amazon_linux "$edition" "$version" "$checksum" "$checksum_type"
   else
-    log_error "This script only supports Ubuntu 16.04 and Amazon Linux 2."
+    log_error "This script only supports Ubuntu 16.04, 18.04, and Amazon Linux 2."
```

Comment:
Sync Gateway has packages for 18.04?

Reply:
Sync Gateway has a single package that isn't indexed by OS. Presumably this works on any distro using apt.